src/main/java/no/ndla/taxonomy/rest/v1/commands/NodePostPut.java

```diff
@@ -58,6 +59,14 @@ public class NodePostPut implements UpdatableDto<Node> {
     @Schema(description = "The language used at create time. Used to set default translation.", example = "nb")
     public Optional<String> language = Optional.empty();
 
+    @JsonProperty
+    @Schema(description = "The quality evaluation of the node. Consist of a score from 1 to 5 and a comment.")
+    @JsonDeserialize(using = QualityEvaluationDTODeserializer.class)
+    @JsonSerialize(using = QualityEvaluationDTOSerializer.class)
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    @NullOrUndefined
```

Comment:
Trengs NullOrUndefined når du har custom serializer?

Reply:
`NullOrUndefined` er bare for å hjelpe typescript genereringen, så ja.
Kunne droppet den og endret på custom mappingen, men da måtte vi eksplisitt sendt `undefined` i fra frontenden, og det har Katrine kjeftet på meg tidligere for 😅